src/App/Game/GameOver.tsx

```diff
@@ -8,10 +8,13 @@ import { Link } from 'react-router-dom';
 import Button from '@material-ui/core/Button';
 import ReplayIcon from '@material-ui/icons/Replay';
 import ReactGA from 'react-ga';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
 
 export interface IGameOverProps {
   result: string;
   gameArgs?: IGameArgs;
+  otherPlayerBoard?: React.ReactNode;
```

Comment:
I like this API (can be any node), but I think the name otherPlayerBoard is not fair, not generic enough. This could be ANY UI component that we want display at game end. I would rename it to be "extraCardContent" or something like this.

Reply:
Good point, thanks!